Prime_Number/Prime_Number.js

```diff
@@ -0,0 +1,33 @@
+//Description: Function to find if a given number is prime or not
+//Expected Output: returns true or false
+
+const isPrime = (num) => {
+  //Numbers less than 2 are not prime numbers (including negatives)
+  if (num < 2) {
```

Comment:
please add 4 space indentation here as well as it is inside the block for function